### PR TITLE
Merge 4.7.0 into master

### PR DESCRIPTION
**Pull request by damarisg**
|Related issue|
|-------------|
|      https://github.com/wazuh/wazuh-qa/issues/4456|

## Description

Merge `4.7.0` into `master`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -79,13 +80,74 @@ Release report: TBD
...
+Wazuh commit: https://github.com/wazuh/wazuh/commit/f6aba151d08ef065dfc1bdc9b8885c3d4f618fca
+Release report: https://github.com/wazuh/wazuh/issues/18235
+
+### Changed
+
+- Delete `update_from_year` from system and E2E tests configuration ([#4372](https://github.com/wazuh/wazuh-qa/pull/4372)) \- (Tests)
+- Upgrade PyYAML to 6.0.1. ([#4326](https://github.com/wazuh/wazuh-qa/pull/4326)) \- (Framework)
+- Change Vulnerability Detector ITs to support the development of the NVD 2.0 refactor. ([#4327](https://github.com/wazuh/wazuh-qa/pull/4327)) \- (Tests)
+
+## [4.5.2] - TBD
```
Comment (QU3B1M) [inline CHANGELOG.md:127]: `## [4.5.2]` and `## [4.5.1]` duplicated, please delete the not updated versions.

Comment (damarisg) [reply CHANGELOG.md]: Fixed https://github.com/wazuh/wazuh-qa/pull/4460/commits/f83c44b3cbabf2a5bc0f4bb2b9a90b6f964dc398

**Review by QU3B1M — APPROVED** [review]
LGTM!

**Comment by damarisg** [comment]
Create a new PR to remove duplicated Changelog https://github.com/wazuh/wazuh-qa/pull/4462

**Pull request merged**